src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -179,9 +179,6 @@ bool RPCServiceImpl::ManageMobileCommand(
       return false;
     }
 #endif  // ENABLE_SECURITY
-
-    // Message for "CheckPermission" must be with attached schema
-    mobile_so_factory().attachSchema(*message, false);
```

Comment:
@YarikMamykin Why did you remove it?

Reply:
@AByzhynar because this one is done in ValidateRPCSO method, that is called inside ConvertMessageToSO. So here we have obsolete second call of attachSchema

Reply:
I believe this is not quite correct, but it does look like this call is covered in the `SendMessageToMobile` function, which should justify this change.